src/tasks/LibraryBuilder/Templates/autoinit.c

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+
+#include <mono/jit/jit.h>
+#include <mono/jit/mono-private-unstable.h>
+#include <mono/metadata/assembly.h>
+
+#ifdef HOST_ANDROID
+#include <android/log.h>
+
+#define LOG_INFO(fmt, ...) __android_log_print(ANDROID_LOG_INFO, "MONO_SELF_CONTAINED_LIBRARY", fmt, ##__VA_ARGS__)
```

Comment:
Do you use LOG_INFO? I think library init must be quite if we don't hit errors.

Reply:
Right now, its only being used to notify users that runtime config file could not be found in case they dont see app context properties they were expecting to exist from the built runtime config(monodroid.c and runtime.m both continue even if its not found). It didnt feel like a scenario in which we should error, but should we?